File: JS/JS_Code_Challenge/Challenge10.js
//Create a function that filters out negative numbers

function negativeNumbers(numbers) {
    return numbers.filter(number  => number >= 0)
}

const numbers = [-23,-20, -12, 5, 0, -1, 5, 12, -19, 20]
const positiveArray = negativeNumbers(numbers)


console.log(positiveArray);

File: Python Coding Challenge /Challenge 10.py
#Create a function that filters out negative numbers.

numbers = [-12, 15, -3, 8, -1, 0, 4, -8, 0.8]

def filter(numbers):
    positive_numbers = [num for num in numbers if num >= 0]
    return positive_numbers

filtered_numbers = filter(numbers)
print("Filtered list:", filtered_numbers)